_src/css/_section.scss

```diff
@@ -45,11 +45,13 @@
     }
 
     .box--links {
-      padding: 20px 0;
-
       @media screen and (min-width: $hd-up) {
         padding: 50px 0;
       }
+      
+      @media screen and (max-width: $hd-up) {
```

Comment:
@felipprodrigues remember that one breakpoint is always for up `(min-width: $hd-up)` and another with 1px for down `(max-width: $hd)`. If you use `$hd-up` twice is not gonna work properly